src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_EyeOfHarmony.java

```diff
@@ -928,9 +934,8 @@ public GT_Multiblock_Tooltip_Builder createTooltip() {
                 .addStructureInfo(
                         "Requires " + EnumChatFormatting.GOLD + 2 + EnumChatFormatting.GRAY + " input hatches.")
                 .addStructureInfo(
-                        "Requires " + EnumChatFormatting.GOLD + 1 + EnumChatFormatting.GRAY + "+ ME output hatch.")
-                .addStructureInfo(
-                        "Requires " + EnumChatFormatting.GOLD + 1 + EnumChatFormatting.GRAY + " input busses.")
+                        "Requires " + EnumChatFormatting.GOLD + "1-18" + EnumChatFormatting.GRAY + " ME output hatch.")
```

Comment:
You no longer need 1-18 me output hatches because now a me output hatch can be used for multiple fluid since https://github.com/GTNewHorizons/TecTech/pull/143